src/lib/web3/wagmi-config.ts

```diff
@@ -0,0 +1,13 @@
+import { getDefaultConfig } from '@rainbow-me/rainbowkit';
+import { Config } from 'wagmi';
+import { mainnet } from 'wagmi/chains';
+
+export const wagmiConfig = getDefaultConfig({
+  appName: 'zOS',
+  projectId: 'TEMP_ID',
```

Comment:
We'll need to set up a project in Wallet Connect Cloud for this. Also will add it to `.env` - have just left it hardcoded for now.